store/store.go

```diff
@@ -1966,6 +1950,7 @@ type SnapAction struct {
 	SnapID       string
 	Channel      string
 	Revision     snap.Revision
+	CohortKey    string
```

Comment:
maybe it comes later but we need this also in context

Reply:
Yep, when I use it (from snapstate)